www/templates/html/SiteMaster/Core/Auditor/Site/Review/View.tpl.php

```diff
@@ -0,0 +1,27 @@
+<?php
+use \SiteMaster\Core\Auditor\Site\Review;
+/**
+ * @var $context \SiteMaster\Core\Auditor\Site\Review\View
+ */
+?>
+
+<?php if (!empty($context->review->result)): ?>
+    <?php
+    $result = '';
+    switch ($context->review->result) {
+        case Review::RESULT_NEEDS_WORK:
+            $result = 'Site needs work';
+            break;
+        case Review::RESULT_OKAY:
+            $result = 'Site is okay';
+            break;
+    }
+    ?>
+    <p>
+        Result: <?php echo $result?>
+    </p>
+<?php endif; ?>
+
+<p>
+    <?php echo $context->review->public_notes ?>
```

Comment:
I assume this will probably evolve, but based on the assumption of a large box of text, the authors will probably want their newlines preserved `nl2br` style.